### PR TITLE
Added meaning of ones in adjacency matrices

### PR DESCRIPTION
**Pull request by harshsaver**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/adjacency-matrix.english.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ forumTopicId: 301621
 <section id='description'>
 
 Another way to represent a graph is to put it in an <dfn>adjacency matrix</dfn>.
-An <dfn>adjacency matrix</dfn> is a two-dimensional (2D) array where each nested array has the same number of elements as the outer array. In other words, it is a matrix or grid of numbers, where the numbers represent the edges. Zeros mean there is no edge or relationship.
+An <dfn>adjacency matrix</dfn> is a two-dimensional (2D) array where each nested array has the same number of elements as the outer array. In other words, it is a matrix or grid of numbers, where the numbers represent the edges. 
+<strong>Note</strong>: The numbers to the top and left of the matrix are just labels for the nodes and the Ones means there exists an edge between the vertices (nodes) representing the row and column and Zeros mean there is no edge or relationship.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/adjacency-matrix.english.md]:
```suggestion

<strong>Note</strong>: The numbers to the top and left of the matrix are just labels for the nodes. Inside the matrix ones mean there exists an edge between the vertices (nodes) representing the row and column.  Finally, zeros mean there is no edge or relationship.
```
One and zero are not a proper nouns - no need to capitalise them.  I also split the sentences to aide comprehension.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/adjacency-matrix.english.md [hunk 2]
```diff
@@ -9,9 +9,10 @@ forumTopicId: 301621
...
 <blockquote>    1 2 3<br>   ------<br>1 | 0 1 1<br>2 | 1 0 0<br>3 | 1 0 0</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/adjacency-matrix.english.md]:
```suggestion
<pre>
    1 2 3
  -------
1 | 0 1 1
2 | 1 0 0
3 | 1 0 0
</pre>
```
Blockquotes don't preserve whitespace, so I changed it to a pre element instead.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks for the PR! I made a few suggestions below.

**Comment by moT01** [comment]
I looks unlikely that the PR creator will make these changes @ojeytonwilliams, do you want to commit them?

**Comment by ojeytonwilliams** [comment]
@raisedadead oops, forgot about this one.  I'll fix the linting.

**Comment by ojeytonwilliams** [comment]
I've fixed the lint problem with a mild hack.  It was interpreting the ----- as a heading, so the slash stops that (and still looks good, imo).

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by moT01 — APPROVED** [review]
LGTM 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**